mitxgraders/formulagrader/matrixgrader.py

```diff
@@ -59,8 +60,23 @@ class MatrixGrader(FormulaGrader):
                         received objects is revealed.
 
         suppress_matrix_messages (bool): If True, suppresses all matrix-related
-            error messages from being displayed. Overrides shape_errors=True and
+            erro2r messages from being displayed. Overrides shape_errors=True and
             is_raised=True. Defaults to False.
+
+        Additionally, the following three configuration keys will be passed to
+        the default comparer and be used to specify partial credit options.
+
+        entry_partial_credit ('proportional'|number): Determines how partial credit
```

Comment:
Rather than repeat all this text, I would just include the entry names and say "See MatrixEqualityComparer". The docstrings aren't for the end users of this library, so anybody who's diving into here can follow that chain.